src/map.h

```diff
@@ -428,7 +428,7 @@ class ServerMap : public Map
 	MapDatabase *dbase = nullptr;
 	MapDatabase *dbase_ro = nullptr;
 
-	MetricCounterPtr m_save_time_counter;
```

Comment:
warning, you have both MetricCounter & MetricGauche in prometheus. If you remove the meaning here, we don't know what is this